### PR TITLE
fix: event card image placeholder

### LINKED ISSUE #2298: Event card image placeholders + spacing
If an event image is not provided, we should subtly highlight the spot where the image would go.

The second event card shows how they look without images
image

Even something as simple as 
image

could work, though I'm sure there are nicer approaches.

Also, could we fix the size of the images?  It seems a little messy to have some larger than others.

Thanks to @Sembauke for pointing out this issue.

### PR DESCRIPTION
**Pull request by atosh502**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

Closes #2298

Results: 
image

### CODE REVIEW

File: client/src/components/EventCard.tsx [hunk 1]
```diff
@@ -88,13 +89,15 @@ export const EventCard: React.FC<EventCardProps> = ({ event }) => {
...
-        src={event.image_url}
-        objectFit={'cover'}
-      />
+      <Box w="240px" h="120px" display={['none', 'block']}>
+        <Image
+          width="100%"
+          height="100%"
+          src={event.image_url}
+          fit="cover"
+          fallback={<ImagePlaceholder />}
```
Comment (Sboonny) [inline client/src/components/EventCard.tsx:98]:
```suggestion
          fallbackSrc="https://cdn.freecodecamp.org/chapter/brown-curtain-small.jpg"
              fallbackStrategy="onError"
```

File: client/src/components/EventCard.tsx [hunk 2]
```diff
@@ -88,13 +89,15 @@ export const EventCard: React.FC<EventCardProps> = ({ event }) => {
...
       {...(hasEnded && { opacity: 0.6 })}
     >
-      <Image
-        display={['none', 'block']}
-        h={'auto'}
-        w={'200px'}
-        src={event.image_url}
-        objectFit={'cover'}
-      />
+      <Box w="240px" h="120px" display={['none', 'block']}>
```
Comment (Sboonny) [inline client/src/components/EventCard.tsx:92]:
```suggestion
      <Box w="240px" h="120px" display={['none', 'block']} background="gray.85">
```

File: client/src/components/EventCard.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { isPast } from 'date-fns';
 import React from 'react';
 import { Chapter, Event } from '../generated/graphql';
 import { formatDate } from '../util/date';
+import ImagePlaceholder from './ImagePlaceholder';
```
Comment (Sboonny) [inline client/src/components/EventCard.tsx:7]:
```suggestion
```

**Review by Sboonny — COMMENTED** [review]
There is an image used as fallback in EventPage, we can use it here as well.

File: client/src/components/EventCard.tsx (hunk 3 above)
Comment (atosh502) [reply client/src/components/EventCard.tsx]: Hi @Sboonny I've updated the code to reuse the existing placeholder.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**